composer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "drupal/islandora_image",
```

Comment:
Should be 'islandora/islandora_image'.  We can't use 'drupal' as the vendor name until we're actually on drupal.org :(